db/schema.rb

```diff
@@ -379,8 +379,9 @@
   end
 
   add_foreign_key "case_logs", "locations"
-  add_foreign_key "case_logs", "schemes"
+  add_foreign_key "case_logs", "organisations", column: "owning_organisation_id", on_delete: :cascade
+  add_foreign_key "case_logs", "schemes", on_delete: :cascade
   add_foreign_key "locations", "schemes"
-  add_foreign_key "schemes", "organisations", column: "managing_organisation_id"
-  add_foreign_key "schemes", "organisations", column: "owning_organisation_id"
+  add_foreign_key "schemes", "organisations", column: "owning_organisation_id", on_delete: :cascade
+  add_foreign_key "users", "organisations", on_delete: :cascade
```

Comment:
Where is this coming from? Looks like there's a migration not committed here?

Reply:
This is fixed